composer/core/state.py

```diff
@@ -82,6 +82,8 @@ class State(Serializable):
         run_name (str): The name for this training run.
         grad_accum (int, optional): The number of gradient accumulation steps to use. With this argument, micro batch
             size for each device becomes ``microbatch_size = train_batch_size / (num_devices * grad_accum)``.
+        eval_batch_split (int, optional): The mirror of grad_accum for eval. With this argument, micro batch
+            size for each device becomes ``microbatch_size = eval_batch_size / (num_devices * grad_accum)``.
```

Comment:
[nit] Could be mistaken but should the denominator have `grad_accum`?  I thought these were independent so `eval_microbatch_size = eval_batch_size // eval_batch_split`?